client/pom.xml

```diff
@@ -182,5 +200,10 @@
             <artifactId>commons-codec</artifactId>
             <version>1.13</version>
         </dependency>
+        <dependency>
+            <groupId>com.vesoft</groupId>
+            <artifactId>nebula-utils</artifactId>
+            <version>1.0.0-beta</version>
```

Comment:
RC2 ?

Reply:
Good catch, wait for `nebula-utils` update